apps/web/.env.example

```
@@ -34,3 +34,24 @@ NEXT_PUBLIC_APP_URL=http://localhost:3000
+SENTRY_ENVIRONMENT=
+
+# 소스맵 업로드 전용 — CI 빌드 서버에만 설정, 절대 커밋 금지
+# Sentry > Settings > Auth Tokens 에서 발급
+SENTRY_AUTH_TOKEN=
+SENTRY_ORG=your-sentry-org-slug
+SENTRY_PROJECT=web
+
+# PII 수집 opt-in — true로 설정 시 요청 헤더·사용자 IP를 Sentry에 전송 (기본값: false)
+# GDPR/CCPA 준수 여부 확인 후 활성화
+SENTRY_SEND_DEFAULT_PII=false
+NEXT_PUBLIC_SENTRY_SEND_DEFAULT_PII=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_SEND_DEFAULT_PII key should go before the SENTRY_SEND_DEFAULT_PII key

(UnorderedKey)